partner_relations_in_tab/model/res_partner_relation_type.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ResPartnerRelationType(models.Model):
+    _inherit = 'res.partner.relation.type'
+
+    own_tab_left = fields.Boolean('Show in own tab', default=False)
+    own_tab_right = fields.Boolean('Show in own tab', default=False)
+
+    def _update_res_partner_fields(self):
```

Comment:
Thinking about performance I really do not like the way this is implemented. With any tab added or deleted, all the tabfields, also those not touched at all, are created from scratch. I should not be to difficult (and probably even less code) to just check for changes in the own_tab_left field or the own_tab_right field, and create/delete the tabfields accordingly.

Reply:
@NL66278 How can this be done? Keep in mind that that `self` here is a recordset not the model so it does not search on all the records, only on those who had their `own_tab_left` and/or `own_tab_right` changed.